README.md

```diff
@@ -3,56 +3,145 @@
 
-## Instructions
+## Changing Our Request With Parameters
+At some point, we will need to alter our request in order to change the data we receive. How we do this depends on the API we are calling but there is a good chance we will need to use url parameters. JQuery provides an easy way to do this by passing our parameters as a JavaScript object. Let's us the Spotify API to search for a song. We will specify 3 different criteria, `q` will be the search string, `type` will be the type of the thing we want to find and `limit` will be how many results we want.
 
-chose an API like [wikipedia](http://www.programmableweb.com/api/wikipedia) or [tumblr](http://www.programmableweb.com/api/tumblr) (do post and get requests) have students code along with the examples - tell them what files to copy and paste
+```js
+var url = 'https://api.spotify.com/v1/search';
+
+// These parameters will be made into url paramters by jQuery
+var url_params = {
+  q: "Take Me to Church",
+  type: "track",
+  limit: 10
+};
+
+var success_callback = function (searchResultsFromAPIRequest){
+  var tracks = searchResultsFromAPIRequest.tracks.items;
+  $.each(tracks, function(index, track) {
+    console.log("Song Found: " + track.name);
+  });
+};
+
+// The second parameter we are passing is the url parameters to use in the request
+$.getJSON(url, url_params, success_callback);
+```
+The result of the request will be the name of all the songs we found printed to the console. Under the covers, jQuery took our paramters and constructed a url to make the request with.
 
-need tests to make sure following along correctly
+```
+https://api.spotify.com/v1/search?qTake+Me+to+Church&type=track&limit10
+```
 
-build methods to return clean info from the API
+Its also possible to create the url ourselves.
 
-have a frontend so students can fill out a form and submit it with the submit event and have the JS make the API call and display the information they want
+```js
+// Here we manually create a url with all the url parameters
+var url = 'https://api.spotify.com/v1/search?q=Take+Me+to+Church&type=track&limit=10';
+
+var success_callback = function (searchResultsFromAPIRequest){
+  var tracks = searchResultsFromAPIRequest.tracks.items;
+  $.each(tracks, function(index, track) {
+    console.log("Song Found: " + track.name);
+  });
+};
+
+// No url parameters being passed since we already included them in the url
+$.getJSON(url, success_callback);
+```
+In the end, we get the same results but we have the option to choose our approach based on the situation. One final thing to consider is that our success callback won't actually be called if the JSON that gets return from the API is invalid. We will need to have a error callback to handle these types of problems.
```

Comment:
The only reason would be that we aren't using jQuery but I figure this isn't a valid statement for this lesson. What do you think.